src/main/java/org/apache/sling/feature/maven/Preprocessor.java

```diff
@@ -404,6 +455,7 @@ public Feature provide(final ArtifactId id) {
 
 	                    // "external" dependency, we can already resolve it
 	                    final File featureFile = ProjectHelper.getOrResolveArtifact(info.project, env.session, env.artifactHandlerManager, env.resolver, id).getFile();
+	                    checkFeatureFileValidation(featureFile);
```

Comment:
I think we should not validate external features as these are released in maven already and therefore should be validated already